docs/tools/vdb_table/data/azureai.json

```diff
@@ -20,9 +20,9 @@
   },
   "dev_languages": {
     "value": [
-      "C#",
-      "C++",
-      "Java"
+      "c#",
+      "c++",
+      "java"
```

Comment:
Hi @farzad528, we are generally constraining this field to the primary language that the Vector DB system has been developed in.

Which of these would you choose as primary?

Reply:
C++